src/redux/reducers/siloDomainData/editEntries.js

```diff
@@ -507,6 +556,21 @@ const setEntryControlStatus = (state, action) => {
     return setEntriesControlStatus(state, newAction);
 };
 
+const setEntryVerificationStatus = (state, action) => {
+    const {
+        leadId,
+        entry,
+    } = action;
+
+    const newAction = {
+        leadId,
+        entries: [entry],
+        updateVersionId: false,
+    };
```

Comment:
@samshara , is this done so that in future you might want to update entries' verification status in bulk?

Reply:
yes. I followed the pattern for control status change.